starwars/src/components/CardComponent.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+// This component will display character information as a card
+
+// class CardComponent extends React.Component {
+//
+//   render(props) {
+//     return (
+//       <div className="card">
+//       <h2>{this.props.name}</h2>
+//       </div>
+//     );
+//   }
+//
+// }
+
+function CardComponent(props) {
+  //ultimately, this function should contain everything to display
+  //the character's information on a single card.
```

Comment:
Here's a fun idea to try:

You can achieve the exact same result if you pass `char={char}` to your CardComponent instead of all of the individual keys on the character object. That would change your `props.gender, props.hair_color,` and so on to `props.char.gender, props.char.hair_color,` and so on. Stay with me, I know you're thinking why would I do that? It's longer syntax. You can destructure!!! :mindblown:

Declare `const { char } = props` between your CardComponent declaration and return statement. It is the same as `char = props.char;` essentially. Then, you can use `char.gender, char.hair_color, etc.` instead. And you can drop the individual character stats you are passing in via props.

Does that make sense? Let me know if I can clarify.